### ISSUE TITLE
Recent activity gives wrong dates

### ISSUE THREAD

**Issue by de-vries** [mapillary/mapillary_issues#634]
Today (15 feb) I've UL-ed 1300 pics. 

On my profile page in the calendar widget, titled 'Recent Activity' it says that I've UL-ed 1300 pics yesterday (14 feb). This happened with every upload as far as I can remember. If I click through to the images, they have the right date though.

**Comment by korst** [comment]
you're fast :) 

My guess was that it had something to do with time zones. btw here is  my profile page: http://www.mapillary.com/profile/de%20vries/uploads

**Comment by de-vries** [comment]
The notification in the front page that says "Last time you were active was <n time> ago" is off by about 17 hours. Probably related.

I'm gonna try to make some pics after 17:00h and see if they count for today.

**Comment by de-vries** [comment]
I uploaded some photos yesterday (8 jun) and as always the date in recent activity was one day off. Today at 00:30 (local, west European time) the one day shift was gone, but this morning at 09:00 it was back.
![image count](https://cloud.githubusercontent.com/assets/1449609/8055197/0e85f18c-0e9e-11e5-8709-e42ab60d5a3e.png)
*Screens taken on 9 june at 00:30 and 09:00.*

**Comment by de-vries** [comment]
I did an experiment with shifting the time of my computer. The result is that only if I put the time of my computer anywhere between 00:00 and 02:00 recent activity gives the correct information. I'm in UTC+02:00 so that made me think it's time zone related.

If I set my time zone to UTC+08:00 I can get the correct information between 00:00 and 08:00.
I didn't test all time zones but this behaviour seems consistent all over the eastern hemisphere  

At UTC-03:00 recent activity is always one day off, but between 21:00 and 00:00 it's 2 days off.
At UTC-06:00 recent activity is always one day off, but between 19:00 and 00:00 it's 2 days off.
At UTC-08:00 recent activity is always one day off, but between 16:00 and 00:00 it's 2 days off.
So, at the western hemisphere there is consistency too.
